pyo3-build-config/src/impl_.rs

```diff
@@ -701,6 +701,7 @@ fn have_python_interpreter() -> bool {
 /// Must be called from a PyO3 crate build script.
 fn is_abi3() -> bool {
     cargo_env_var("CARGO_FEATURE_ABI3").is_some()
+        || env_var("PYO3_USE_ABI3_FORWARD_COMPATIBILITY").map_or(false, |os_str| os_str == "1")
```

Comment:
Personally, I would just check whether the variable is present or not. Do we have precedence for the check for `1` specifically?

Reply:
I slightly prefer being explicit, and this is consistent with what we did for `PYO3_PRINT_CONFIG`. If you have a strong preference I can relax both?

Reply:
No, it is fine if we did it like this elsewhere.